package.json

```diff
@@ -41,9 +41,27 @@
     "xmlbuilder": "15.1.1"
   },
   "devDependencies": {
+    "@babel/core": "7.12.10",
+    "@babel/preset-env": "7.12.11",
+    "autoprefixer": "10.1.0",
+    "babel-loader": "8.2.2",
     "babel-plugin-istanbul": "6.0.0",
     "buffer": "6.0.0",
+    "commitizen": "4.2.2",
+    "css-loader": "5.0.1",
     "dotenv": "8.2.0",
+    "eslint": "7.17.0",
+    "eslint-config-airbnb": "18.2.1",
+    "eslint-config-prettier": "7.1.0",
+    "eslint-import-resolver-webpack": "0.13.0",
+    "eslint-loader": "4.0.2",
+    "eslint-plugin-import": "2.22.1",
+    "eslint-plugin-jsx-a11y": "6.4.1",
+    "eslint-plugin-prettier": "3.3.0",
+    "eslint-plugin-react": "7.22.0",
+    "hson-loader": "2.0.0",
```

Comment:
Do we use hson somewhere?

Reply:
I think this is used in pdb loading when running tests. Installed it when `npm test` complained.